src/components/Property/Property.js

```diff
@@ -69,14 +77,13 @@ export default class Property extends PureComponent {
           {isClickable && <Indicator className={classes.indicator} direction={indicatorDirection} />}
         </td>
         <td className={classes.info}>
-          {title && <span className={classes.title}>{title}</span>}
           <span className={classes.type}>
             {!subtype ? type.join(', ') : <span className={classes.subType}>{subtype}[]</span>}
             {!subtype && constraints && constraints.format &&
             <span className={classes.format}>&lt;{constraints.format}&gt;</span>}
           </span>
           <PropertyConstraints constraints={constraints} type={type} isRequired={isRequired} />
-          {(enumValues || defaultValue || description) &&
+          {((enumValues && enumValues.length > 0) || defaultValue || description) &&
```

Comment:
FYI this can be simplified to `enumValues && enumValues.length`. The reason it works is because 0 will become `false`, anything else will be treated as `true`.